src/customFileParser/unpackGeometry/InstancedMesh.ts

```diff
@@ -19,16 +19,16 @@ export default function unpackInstancedMeshes(
   ) {
 
   const data = new PropertyLoader(uncompressedValues);
-  const meshCounts: {[fileId: string]: {[triangleOffset: string]: { count: number, triangleCount: number }}} = {};
-  const fileIdToSector: {[fileId: string]: Sector} = {};
 
-  // Count meshes per file Id and triangle offset
-  for (const sector of rootSector.traverseSectors()) {
+  for (const sector of rootSector!.traverseSectors()) {
```

Comment:
Why did anything change here? rootSector is not an optional argument into the function.

Reply:
A case of bad copy/paste. Fixed.